hs_collection_resource/tests/test_collection.py

```diff
@@ -694,11 +680,10 @@ def test_are_all_contained_resources_published(self):
         self.assertEqual(self.resCollection.are_all_contained_resources_published, False)
 
         self.assertEqual(self.resGen1.raccess.published, False)
-        self.assertEqual(self.resGeoFeature.raccess.published, False)
 
         # add 2 unpublished resources to collection
         self.resCollection.resources.add(self.resGen1)
-        self.resCollection.resources.add(self.resGeoFeature)
+        self.resCollection.resources.add(self.resTimeSeries)
```

Comment:
Updating everything from geofeature to timeseries means that this will have to be updated again when TimeSeriesResource type is removed.